rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -1178,7 +1178,7 @@ void ppu_load_exec(const ppu_exec_object& elf)
 				{
 					sdk_version = info.sdk_version;
 					primary_prio = info.primary_prio;
-					primary_stacksize = info.primary_stacksize;
+					primary_stacksize = std::min<u32>(info.primary_stacksize, 0x100000);
```

Comment:
use std::clamp here with 0x8000 as min and 0x100000 as max

Reply:
That would break the switch at https://github.com/rpcs3/rpcs3/blob/master/rpcs3/Emu/Cell/PPUModule.cpp#L1466

Perhaps doing a clamp after the switch case would be optimal.

Reply:
Oh I already thought this was after the switch. Yeah you should definitely move the clamp to right after the switch.